daraja/src/main/java/com/github/daraja/driver/DarajaDriver.kt

```diff
@@ -94,10 +96,10 @@ class DarajaDriver(private val consumerKey: String, private val consumerSecret:
             else -> {}
```

Comment:
Is there a use for this else statement ?

Reply:
Not really but I can remove it

Reply:
The `when` clause is complaining so I have to leave it that way